cypress/e2e/default/learn/challenges/lower-jaw.ts

```diff
@@ -0,0 +1,32 @@
+const outputLocation = {
+  catApp:
+    '/learn/2022/responsive-web-design/' +
+    'learn-html-by-building-a-cat-photo-app/step-2'
+};
+
+describe('lower jaw', () => {
+  before(() => {
+    cy.visit(outputLocation.catApp);
+  });
+
+  it(
+    'Should show the quote when the code passes',
+    { browser: 'electron' },
```

Comment:
Why only run this on electron?

Reply:
I was following a similar pattern I found in `projects` cypress test. I've updated to include more browsers in the fix (`'electron', 'chrome', 'chromium', 'firefox', 'edge'`).

Reply:
I just removed them all-together.